src/agent/structures/structures.ts

```diff
@@ -73,10 +79,11 @@ export interface ListThreadsParameters {
 }
 
 export interface ListArchivesParameters {
-  filters?: ArchivesFilters;
+  filters: ArchivesFilters;
```

Comment:
```suggestion
  filters?: ArchivesFilters;
```
It should be optional like in newer versions

Reply:
OK in v3.5